packages/host/app/components/ai-assistant/card-pill.gts

```diff
@@ -0,0 +1,40 @@
+import Component from '@glimmer/component';
+
+import Pill from '@cardstack/host/components/pill';
+
+import { type CardDef } from 'https://cardstack.com/base/card-api';
+
+interface Signature {
+  Element: HTMLElement;
+  Args: { card: CardDef };
+  Blocks: { default: [] };
+}
+
+export default class AiAssistantCardPill extends Component<Signature> {
+  <template>
+    {{#if @card.id}}
+      <Pill @inert={{true}} class='card-pill' ...attributes>
+        <div class='card-title'>{{getDisplayTitle @card}}</div>
```

Comment:
Is there a separate task to add the realm icon to this?

Reply:
Also, should we use the atom format of the card here instead of `getDisplayTitle`?

Reply:
I added the icon to the input attachments side, I can create a ticket to handle it for the atom template. Does this mean that atom template should have the realm icon by default? Also the atom styling for card templates is slightly different from how we previously used them, but I can update them too